packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -385,14 +411,18 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
         break;
 
       case KeyCodes.up:
-        this.setSelectedIndex(this.state.selectedIndex - 1);
+        newIndex = this._moveIndex(this.state.selectedIndex - 1);
```

Comment:
Hey @dzearing, made some changes to the method.  I think this may be a better solution cause it solves keypress home/end as well, instead of changing the selectedIndex by 1 for just the up/down keypress.  What do you think? 

Also, we don't want to check against Normal type, since options won't show that unless defined explicitly.